oom-frontend/src/app/site/pages/auth/signin/signin.component.ts

```diff
@@ -8,11 +10,22 @@ import {SigninInfoDTO} from '../../../../shared/models/dto/SigninInfoDTO.model';
 })
 export class SigninComponent implements OnInit {
 
-  constructor() { }
+  constructor(private authService: AuthService, private router: Router) { }
 
   form = new SigninInfoDTO();
+  loginFail = false;
 
   ngOnInit(): void {
   }
 
+  submit(): void {
+    this.authService.login(this.form).subscribe(res => {
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/Out-Of-Memory-Team/out-of-memory/pullRequest?prid=7533964)
```suggestion
    this.authService.login(this.form).subscribe((res) => {
```